net/modemmanager/README.md

```diff
@@ -34,3 +37,12 @@ allowing all protocols.
 
 The 'iptype' option supports any of these values: 'ipv4', 'ipv6' or 'ipv4v6'.
 It will default to 'ipv4' if not given.
+
```

Comment:
You're mixing device-level options (e.g. signalrate) with daemon-level options (debugmode and loglevel). If you only have one single modem, this may work, but it won't work ifyou have multiple modems with one settings section for each. You cannot have daemon-level options in `/etc/config/network`